### PR TITLE
Allow the reporter method used by reviewdog to be configured

### PR DESCRIPTION
**Pull request by K-Phoen**
Hey!

In this pull request, I allow the `reporter` used by reviewdog to be configured.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,6 +3,10 @@ FROM alpine:3.10
 RUN wget -O - -q https://raw.githubusercontent.com/reviewdog/reviewdog/master/install.sh| sh -s -- -b /usr/local/bin/ v0.9.13
 RUN wget -O - -q https://git.io/misspell | sh -s -- -b /usr/local/bin/
 
+RUN apk --update add git && \
+    rm -rf /var/lib/apt/lists/* && \
+    rm /var/cache/apk/*
```
Comment (thinca) [inline Dockerfile]: What about [`--no-cache`](https://github.com/gliderlabs/docker-alpine/blob/master/docs/usage.md#disabling-cache) ?

```dockeerfile
RUN apk --no-cache add git
```

Comment (K-Phoen) [reply Dockerfile]: Done :)

**Comment by thinca** [comment]
@haya14busa I want to use this for private repository!

**Review by haya14busa — APPROVED** [review]
Thanks!

**Comment by haya14busa** [comment]
Released v1.1.0 & new v1

**Pull request merged**